### ISSUE TITLE
Can't override getter - Maximum call stack size exceeded 

### ISSUE THREAD

**Issue by jurgens** [asciidisco/Backbone.Mutators#30]
```
Test = Backbone.Model.extend

  defaults:
    name: ''

  mutators:
    name: ->
      @get('name') + ' mutated'

test = new Test
#> Maximum call stack size exceeded 
```

**Comment by danielmcormond** [comment]
@asciidisco I believe the issue is still present in vanilla JavaScript. I've put together [a simple JSFiddle here](http://jsfiddle.net/xtL984h4/) to demonstrate. The code is a straight copy-and-paste from the [Backbone.Mutators docs](https://github.com/asciidisco/Backbone.Mutators#override-getters).

It throws an `Uncaught RangeError: Maximum call stack size exceeded` because it just keeps calling the mutated getter recursively forever.

Maybe we could pass in the original Backbone getter as an argument to use within the mutated getter? For example, similar to how it's [currently done with setters](https://github.com/asciidisco/Backbone.Mutators#use-mutated-setters-and-call-the-original-setter-within).

**Comment by maximerassi** [comment]
+1 same issue

**Comment by HiroAgustin** [comment]
Same here

**Comment by passabilities** [comment]
@RasCarlito 's solution works for me +1

**Comment by cnobile2012** [comment]
Yup, same here. Still broken. @RasCarlito 's solution does work.
